kong/init.lua

```diff
@@ -382,6 +385,9 @@ function Kong.init()
     end
   end
 
+  local clustering = require "kong.clustering"
+  clustering.init(config)
```

Comment:
Why is this being initialized twice in this function?

Reply:
I think this was a rebase error. Looking.